client/i18n/locales/english/intro.json

```diff
@@ -2057,46 +2107,82 @@
         "intro": [
           "For this lab, you will debug a random background color changer and fix the errors to make it work properly."
         ]
       },
+      "xeqa": { "title": "205", "intro": [] },
+      "quiz-debugging-javascript": {
+        "title": "Debugging JavaScript Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on Debugging JavaScript."
+        ]
+      },
       "dqth": { "title": "207", "intro": [] },
       "ilop": { "title": "208", "intro": [] },
       "qrer": { "title": "209", "intro": [] },
       "unyy": { "title": "210", "intro": [] },
       "svsl": { "title": "211", "intro": [] },
-      "wnmj": { "title": "212", "intro": [] },
+      "quiz-javascript-regular-expressions": {
+        "title": "JavaScript Regular Expressions Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on JavaScript Regular Expressions."
+        ]
+      },
       "zalp": { "title": "213", "intro": [] },
       "hhvp": { "title": "214", "intro": [] },
       "egkv": { "title": "215", "intro": [] },
       "qlvw": { "title": "216", "intro": [] },
-      "cyrl": { "title": "217", "intro": [] },
+      "quiz-form-validation-with-javascript": {
+        "title": "Form Validation with Javascript Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on Form Validation with Javascript."
+        ]
+      },
       "lupt": { "title": "218", "intro": [] },
       "yvqn": { "title": "219", "intro": [] },
       "xtfk": { "title": "220", "intro": [] },
       "xnpr": { "title": "221", "intro": [] },
-      "lvts": { "title": "222", "intro": [] },
+      "quiz-javascript-dates": {
+        "title": "JavaScript Dates Quiz",
+        "intro": ["Test what you've learned in this quiz on JavaScript Dates."]
+      },
       "foal": { "title": "223", "intro": [] },
       "crzf": { "title": "224", "intro": [] },
       "xofr": { "title": "225", "intro": [] },
       "bsmn": { "title": "226", "intro": [] },
-      "pehm": { "title": "227", "intro": [] },
+      "quiz-javascript-audio-and-video": {
+        "title": "JavaScript Audio and Video Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on JavaScript Audio and Video."
+        ]
+      },
       "cvsw": { "title": "228", "intro": [] },
       "qjws": { "title": "229", "intro": [] },
       "fgbp": { "title": "230", "intro": [] },
-      "jbst": { "title": "232", "intro": [] },
+      "quiz-local-storage-and-crud": {
+        "title": "Local Storage and CRUD Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on Local Storage and CRUD."
+        ]
+      },
       "peyf": { "title": "233", "intro": [] },
       "bdqj": { "title": "234", "intro": [] },
       "ndlf": { "title": "235", "intro": [] },
       "cpmq": { "title": "236", "intro": [] },
-      "yshh": { "title": "237", "intro": [] },
+      "quiz-javascript-recursion": {
+        "title": "JavaScript Recursion Quiz",
```

Comment:
I wonder if this should just be recursion instead of javascript recursion

Reply:
✅ added with next commit.